[filebeat][azure-blob-storage] Added support for more mime types, offset tracking via cursor (#33981)
elastic/beats: x-pack/filebeat/docs/inputs/input-azure-blob-storage.asciidoc

    describing said error.

[id="supported-types"]

> NOTE: NOTE: Currently only `JSON` and `NDJSON` are supported blob/file formats. Blobs/files may be also be gzip compressed.

Reviewer: ```suggestion
NOTE: Currently only `JSON` and `NDJSON` are supported blob/file formats. Blobs/files may be also be gzip compressed.
```

> NOTE: Currently only `JSON`, `NDJSON` are supported with respect to blob/file formats. Blobs/files may be also be gzip compressed.

Reviewer: ```suggestion
NOTE: Currently only `JSON` and `NDJSON` are supported blob/file formats. Blobs/files may be also be gzip compressed.
```
